widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java

```diff
@@ -30,7 +31,7 @@
  * Before this activity is launched, make sure that Glia Widgets SDK is set up correctly.
  * <p>
  * Data that can be passed together with the Activity intent:
- * - {@link GliaWidgets#QUEUE_ID}: ID of the queue you would like to use for your engagements.
+ * - {@link GliaWidgets#QUEUE_IDS}: IDs of the queues you would like to use for your engagements.
```

Comment:
```suggestion
 * - {@link GliaWidgets#QUEUE_IDS}: IDs list of the queues you would like to use for your engagements.
```

Reply:
It is a bit confusing since in other places it is a list and an array. Would it be reasonable to change it to list in the `Intent` parcele?

Reply:
It is always a `List<String>`. But it should be converted to `ArrayList<String>` to pass it to the `Intent`.